awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
Judging by name, I would expect this method is only for setting element properties/attributes. But it also adds elements to DOM sometimes https://github.com/smargh/awesomplete/blob/pr/dropdown/awesomplete.js#L316-L323

@LeaVerou I wonder, if there is really a need for a public API method, which only adds properties/attributes to the element (or also creates DOM nodes sometimes)?
What if this will be an internal helper, instead of a new public API method?